projects/storefrontlib/src/lib/product/components/product-details/container/product-details.component.ts

```diff
@@ -15,15 +16,15 @@ import { ProductDetailOutlets } from '../../../product-outlets.model';
 @Component({
   selector: 'cx-product-details',
   templateUrl: './product-details.component.html',
-  styleUrls: ['./product-details.component.scss'],
+  styleUrls: [
+    './product-details.component.scss',
+    './product-details-tabs.component.scss'
```

Comment:
Not sure the tabs file is needed, it contains no styling.

Reply:
I've also removed the incorrect view encapsulation - it was a left over